dev/docker/Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY . .
 
 # Build the final node binary
 ARG GIT_COMMIT=unknown
-RUN go build -ldflags="-X 'main.Commit=$GIT_COMMIT'" -o bin/xmtpd cmd/xmtpd/main.go
+RUN go build -tags="gowaku_no_rln" -ldflags="-X 'main.Commit=$GIT_COMMIT'" -o bin/xmtpd cmd/xmtpd/main.go
```

Comment:
The docker build was failing with this:
```
44.72 # command-line-arguments
44.72 /usr/local/go/pkg/tool/linux_arm64/link: running gcc failed: exit status 1
44.72 /usr/lib/gcc/aarch64-alpine-linux-musl/12.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/waku-org/go-zerokit-rln-arm@v0.0.0-20230914230828-f14923ed4dac/rln/../libs/aarch64-unknown-linux-gnu/librln.a(std-d6de9543d35dcb36.std.0c288059-cgu.0.rcgu.o): in function `std::sys::unix::os::glibc_version':
44.72 /rustc/69f9c33d71c871fc16ac445211281c6e7a340943/library/std/src/sys/unix/os.rs:660: undefined reference to `gnu_get_libc_version'
44.72 /usr/lib/gcc/aarch64-alpine-linux-musl/12.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/waku-org/go-zerokit-rln-arm@v0.0.0-20230914230828-f14923ed4dac/rln/../libs/aarch64-unknown-linux-gnu/librln.a(std-d6de9543d35dcb36.std.0c288059-cgu.0.rcgu.o): in function `std::sys::unix::net::on_resolver_failure':
44.72 /rustc/69f9c33d71c871fc16ac445211281c6e7a340943/library/std/src/sys/unix/net.rs:528: undefined reference to `__res_init'
44.72 collect2: error: ld returned 1 exit status
```

Noticed a flag to disable rln compilation, so it was the easy solution until it's needed